main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
I'm not sure about this approach. It was a no brainer option for now, the correct way it to named it main_test, but I wouldn't be able to test private functions inside main package.